### PR TITLE
feat(route): add scau/yjsy 

### PR DESCRIPTION
**Pull request by Chunssu**
yjsy.js title and description
华南农业大学研究生院 通知公告

## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/scau/yjsy
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [x] EN / 英文文档
  - [x] CN / 中文文档
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/scau/yjsy.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        title: '华南农业大学研究生院',
+        link,
+        description: '通知公告',
+        item:
+            list &&
+            list.toArray().map((item) => {
+                item = $(item);
+                const a = item.find('a');
+                return {
+                    title: a.text(),
```
Comment (TonyRL) [inline lib/v2/scau/yjsy.js:21]: The selector includes everything as you can see from the first item of https://github.com/DIYgod/RSSHub/pull/13455#issuecomment-1744310879

File: website/docs/routes/university.md [hunk 2]
```diff
@@ -1833,6 +1833,10 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 <Route author="shengmaosu" example="/scau/yzb" path="/scau/yzb" radar="1"/>
 
+### 研究生院通知 {#hua-nan-nong-ye-da-xue-yan-jiu-sheng-yuan-tong-zhi}
+
+<Route author="Chunssu" example="/scau/yjsy" path="/scau/yjsy" radar="1"/>
```
Comment (TonyRL) [inline website/docs/routes/university.md:1838]: Missing corresponding entry in radar.js

File: lib/v2/scau/yjsy.js (hunk 1 above)
Comment (Chunssu) [reply lib/v2/scau/yjsy.js:21]: Thank your comment. I have found my mistake. And I have corrected it.

File: website/docs/routes/university.md (hunk 2 above)
Comment (Chunssu) [reply website/docs/routes/university.md:1838]: Thank you for your comments. I have supplemented entry in radar.js

**Pull request merged**